ext/serializer.c

```diff
@@ -1202,6 +1202,11 @@ void ddtrace_shutdown_span_sampling_limiter(void) {
     zend_hash_destroy(&dd_span_sampling_limiters);
 }
 
+static zend_always_inline double ddtrace_convert_string_to_double(zend_string *str) {
```

Comment:
There's no possibility for that use case to be something else than `true`? I ask because [we do something more comprehensive in .NET](https://github.com/DataDog/dd-trace-dotnet/blob/dbee006bb81c3a5f724d39f9265ac754242f17fe/tracer/src/Datadog.Trace/Activity/OtlpHelpers.cs#L429)

Reply:
Indeed. I re-read the RFC, and it must match the same behavior as [strconv.ParseBool](https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/strconv/atob.go;l=10). I'll change this as soon as I can.

Note: It's not tested in the system tests right now. I may add them.

Reply:
Thanks a lot for the thorough follow up!